src/objects.rs

```diff
@@ -103,6 +103,38 @@ impl OsmId {
             _ => None,
         }
     }
+    /// Returns the inner id.
+    pub fn as_i64(&self) -> i64 {
```

Comment:
```rust
pub fn id(&self) -> i64 {
```

Reply:
`id.id` seemed a bit weird to me...

Reply:
`inner_id`? I don't want `as` in the name of the method as you lost information in the conversion.

Reply:
Let's go for `inner_id` then!